src/java-runtime/java-runtime.targets

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<ItemGroup>
+    <_RuntimeOutput Include="$(OutputPath)java_runtime.jar">
```

Comment:
Indentation in this file is "wonky"; it should be 2 space indents, not all lines are indented properly, etc.